### PR TITLE
fix: silent error

### LINKED ISSUE #37: v3.0.0 does not work
- behavior: output empty styles
- node: v10.16.0
- framework: next.js (v9.5.3)
- plugins: styled-jsx-plugin-sass (v1.0.0)

I tested the version using v2.x and it worked
Maybe related to #35 commit

### PR DESCRIPTION
**Pull request by ramasilveyra**
Fix silent error, this can happen with invalid postcss config.

Fix https://github.com/giuseppeg/styled-jsx-plugin-postcss/issues/37

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -17,7 +17,11 @@ module.exports = (css, settings) => {
 
   processedCss = result.stdout
 
-  if (processedCss instanceof Error || processedCss.name === 'CssSyntaxError') {
+  if (result.stderr) {
+    throw new Error(`postcss failed with ${result.stderr}`)
```
Comment (giuseppeg) [inline index.js]: Would it make sense to append a paragraph with a link to https://err.sh/next.js/postcss-shape ?

File: index.js [hunk 2]
```diff
@@ -17,7 +17,11 @@ module.exports = (css, settings) => {
...
+  }
+
+  if (processedCss instanceof Error || processedCss.name === 'CssSyntaxError' ) {
```
Comment (giuseppeg) [inline index.js]: Now that we use a child process this is never going to be the case and should be moved to `processor.js`

**Review by giuseppeg — COMMENTED** [review]
Thanks a lot for the PR @ramasilveyra !! Just a couple of comments

File: index.js (hunk 1 above)
Comment (ramasilveyra) [reply index.js]: Totally! Done

File: index.js (hunk 2 above)
Comment (ramasilveyra) [reply index.js]: Good catch, fixed!

**Comment by giuseppeg** [comment]
@ramasilveyra ❤️ thank you again for your contribution!

**Pull request merged**